core/src/main/java/com/google/common/truth/IterableSubject.java

```diff
@@ -306,16 +307,21 @@ public final Ordered containsAtLeastElementsIn(Iterable<?> expectedIterable) {
      * actual iterable than the default of "but was," which may _sound_ like it should show only the
      * required elements, rather than the full actual iterable.
      */
-    return ordered
-        ? IN_ORDER
-        : new Ordered() {
-          @Override
-          public void inOrder() {
-            failWithActual(
-                simpleFact("required elements were all found, but order was wrong"),
-                fact("expected order for required elements", expected));
-          }
-        };
+    if (ordered) {
+      return IN_ORDER;
+    } else {
+      order.retainAll(expected);
+      return new Ordered() {
+        @Override
+        public void inOrder() {
+          failWithActual(
```

Comment:
`failWithActual()` will still append a `but was` fact at the end with the full contents of `actual`. That would make the error message extremely confusing and grammatically incorrect.

Reply:
OK，I will replace `failWithActual()` with `failWithoutActual()`